### PR TITLE
opercmds: add chghost, chgident, chgname commands

### PR DESCRIPTION
**Pull request by cooper**
#481 

I made all 3 of these use the `opercmds.chghost` priv, but it would be easy to separate them if desired.

### CODE REVIEW

**Comment by GLolol** [comment]
I prefer having separate permissions for these for the sake of consistency. Other than that, it looks fine.

File: plugins/opercmds.py [hunk 1]
```diff
@@ -204,3 +204,42 @@ def topic(irc, source, args):
     irc.call_hooks([irc.pseudoclient.uid, 'CHANCMDS_TOPIC',
                    {'channel': channel, 'text': topic, 'setter': source,
                     'parse_as': 'TOPIC'}])
+
+@utils.add_cmd
+def chghost(irc, source, args):
+    """<user> <new host>
+
+    Admin only. Changes the visible host of the target user."""
+    chgfield(irc, source, args, 'host')
```
Comment (GLolol) [inline plugins/opercmds.py]: Just hardcode the field argument in uppercase and remove the `field.upper()` call later on

File: plugins/opercmds.py [hunk 2]
```diff
@@ -204,3 +204,42 @@ def topic(irc, source, args):
...
+        target = args[0]
+        new = args[1]
+    except IndexError:
+        irc.error("Not enough arguments. Needs 2: target, new %s." % field)
+        return
+    
+    # Find the user
+    targetu = irc.nick_to_uid(target)
+    if targetu not in irc.users:
+        irc.error("No such nick '%s'." % target)
```
Comment (GLolol) [inline plugins/opercmds.py]: Use `%r` instead of `'%s'`

File: plugins/opercmds.py (hunk 1 above)
Comment (cooper) [inline plugins/opercmds.py]: that would cause the parameter help text to show HOST and IDENT in caps

**Comment by cooper** [comment]
split up permissions

File: plugins/opercmds.py (hunk 1 above)
Comment (GLolol) [inline plugins/opercmds.py]: Fair enough

**Pull request merged**